example_remoteSign_test.go

```diff
@@ -0,0 +1,70 @@
+package notation_test
+
+import (
+	"context"
+	"crypto/x509"
+	"fmt"
+
+	_ "github.com/notaryproject/notation-core-go/signature/cose"
+	_ "github.com/notaryproject/notation-core-go/signature/jws"
+	"github.com/notaryproject/notation-core-go/testhelper"
+	"github.com/notaryproject/notation-go"
+	"github.com/notaryproject/notation-go/registry"
+	"github.com/notaryproject/notation-go/signer"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"oras.land/oras-go/v2/registry/remote"
+)
+
+var targetDesc ocispec.Descriptor
```

Comment:
This line seems redundant.

Reply:
I see. Can we declare anything else other than `var targetDesc ocispec.Descriptor`?